src/realtime/handlers.ts

```diff
@@ -156,8 +164,14 @@ export class RealtimeSocketHandler {
         this.sub?.onInfo?.(data as Info);
         break;
 
+      case MessagesEnum.StartRecognition:
+      case MessagesEnum.AddAudio:
+      case MessagesEnum.EndOfStream:
+      case MessagesEnum.SetRecognitionConfig:
+        // TODO: is this correct?
```

Comment:
@mnemitz not sure of the history of this bit of code, seems like we aren't handling messages that should be handled? Perhaps the messages were added since this was written?

Reply:
If I understand correctly, these message types are ones which we expect to be sending, but not receiving. But since we only have one enum type for all messages, this fact isn't captured in the types.